apps/e2e/lookup-storybook/src/app/lookup/lookup.component.stories.ts

```diff
@@ -0,0 +1,33 @@
+import { Meta, StoryFn, moduleMetadata } from '@storybook/angular';
+
+import { LookupComponent } from './lookup.component';
+import { LookupModule } from './lookup.module';
+
+export default {
+  id: 'lookupcomponent-lookup',
+  title: 'Components/Lookup',
+  component: LookupComponent,
+  decorators: [
+    moduleMetadata({
+      imports: [LookupModule],
+    }),
+  ],
+} as Meta<LookupComponent>;
+const Template: StoryFn<LookupComponent> = (args: LookupComponent) => ({
```

Comment:
Minor - but in most places we call this constant the name of the component. I know we haven't in some of the places where we do multiple stories - but could we call this `Lookup` instead of `Template` to match what they non-dynamic ones do?

Reply:
since it does still refer to the component and not lookup itself, do we want to be more descriptive with the name and call it `BaseLookupComponent` or `BaseLookup` or something?

Reply:
I would do `BaseLookup` if you don't want to do `Lookup`. 

We did `Error` in a similar case - but I don't feel strongly enough to hold it up between the two.